lms/static/sass/_ie.scss

```diff
@@ -40,8 +37,8 @@
   }
 
   // because ie doesn't like :last
-  .last {
-    margin-right: 0  !important;
+  .last.last {
```

Comment:
Good question - we can't use the ``!important`` flag with Sass mixin syntax (which is what I'm changing this to for RTL support). I'm using another method ([a double class method](http://csswizardry.com/2014/07/hacks-for-dealing-with-specificity/)) to make sure this rule trumps others from a CSS specificity PoV.